languages/elixir/exercises/concept/errors/.docs/introduction.md

```diff
@@ -0,0 +1,40 @@
+Errors happen. In Elixir, while people often say to "let it crash", there are times when we need to rescue the function call to a known good state to fulfil a software contract. In some languages, errors are used as method of control flow, but in Elixir, this pattern is discouraged. We can often recognize functions that may rause an error just by their name: functions that raise errors are to have `!` at the end of their name. This is in comparison with functions that return `{:ok, value}` or `:error`. Look at these library examples:
```

Comment:
```suggestion
Errors happen. In Elixir, while people often say to "let it crash", there are times when we need to rescue the function call to a known good state to fulfil a software contract. In some languages, errors are used as method of control flow, but in Elixir, this pattern is discouraged. We can often recognize functions that may raise an error just by their name: functions that raise errors are to have `!` at the end of their name. This is in comparison with functions that return `{:ok, value}` or `:error`. Look at these library examples:
```

Reply:
I hope in `after.md` we could explain better what it means "to use errors for control flow" so that people know exactly what to avoid and why.

Reply:
I'm not sure if GitHub showed well what my suggested change did. It was to fix the typo "rause" -> "raise", and the typo is still there.